### PR TITLE
working on randomizing order

### PR DESCRIPTION
**Pull request by mattk5**

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -14,4 +14,17 @@ def create
       redirect_to "/games/new"
     end
   end
+
+  def update
+    user = current_user
+    users = []
+    user.game.users.all.each do |user|
+      users =
```
Comment (anderskraken) [inline app/controllers/users_controller.rb]: Hey matt, this line looks a bit funky?

File: app/views/games/new.html.erb [hunk 2]
```diff
@@ -1,10 +1,15 @@
 <h3>Your game code is:</h3>
 <h4><%= current_user.game.key_code %></h4>
-<h3><%= current_user.name %></h3>
+<h3>Name: <%= current_user.name %></h3>
+
+<h5>Current number of players: <%= current_user.game.users.length%></h5><br>
+
+<%= %>
```
Comment (anderskraken) [inline app/views/games/new.html.erb]: assuming we don't need this line?

File: app/views/games/new.html.erb [hunk 3]
```diff
@@ -1,10 +1,15 @@
...
+
+
 
-<h5><%= current_user.game.users.length%> Players have joined</h5>
 <% if current_user.creator == true %>
-<form method="POST" action="/games">
-<input type="submit" value="Start game">
+<form method="PUT" action="/games/:game_id/users/">
```
Comment (anderskraken) [inline app/views/games/new.html.erb:12]: don't we need to put a PUT method in hidden value? to a POST method?

**Review by anderskraken — CHANGES_REQUESTED** [review]
A few suggestions for you to review.

**Review by mattk5 — COMMENTED** [review]
commented things out and mades notes.

**Review by anderskraken — APPROVED** [review]

**Pull request merged**